src/python/grapl_e2e_tests/tests.py

```diff
@@ -232,7 +232,7 @@ def check_login() -> None:
 def check_invalid_creds() -> None:
     resp = EngagementEdgeClient().invalid_creds()
     if resp.status_code == 403:
-        assert "Provided invalid creds & was unauthorized"
+        assert resp.status_code == 403, "Provided invalid creds which were unauthorized"
```

Comment:
`if X: assert X` smells fishy.

Sounds like you just want to tell the end-user "hey, there is an error here", which is not what `assert` is for. 
Assert is for, well, asserting "the following must be true." 

The following seems more correct:
```raise Exception("Provided invalid creds, which were unauthorized")```

Reply:
I think I wanted assert here because my `invalid_credentials` method is expecting a 403 to be returned, I don't  think I want to raise an exception here because I wanted to test if we provide invalid credentials, we should get an response noting that. I don't think we're testing for an exception here, what do you think. Basically, if we get a 403 with invalid creds, I want that test to pass because that's the expected behavior for this test. I think I'd want an exception if we provided valid creds with and got a 403